.github/workflows/javadoc.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        java-version: [11, 17]
+        java-version: [11, 17, 21]
```

Comment:
This feels excessive. We can just do 21 (latest) or 11 (since that is current java version)

Reply:
I guess let's go with 11 then.